packages/dds/tree/src/feature-libraries/sequence-change-family/changeset/format.ts

```diff
@@ -107,6 +107,16 @@ export namespace Transposed {
          * Indicates a prior concurrent slice-delete that the target place was affected by.
          */
         scorch?: PriorOp;
+
+        /**
+         * Record of relevant information about changes this mark has been rebased over.
+         */
+        lineage?: LineageEvent[];
+    }
+
+    export interface LineageEvent {
+        readonly revision: RevisionTag;
+        readonly offset: number;
```

Comment:
The sequence-change-family changeset is mostly dead code being kept because it has code and test cases that have yet to be migrated to the sequence-field implementation and tests. Feel free to drop the changes to this file.